.github/workflows/snyk_sca_scan.yml

```diff
@@ -0,0 +1,37 @@
+name: Snyk Code Security
+
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  open-source:
```

Comment:
Instead of a new file, I think you should just add this as another step to the existing CI file.

Reply:
As we talked about, we planned to make the Snyk scan controller through a GitHub action in Q3. Hence, let's separate this file from the CI file.